### PR TITLE
These are some changes from RUB-290

### PR DESCRIPTION
**Pull request by ishwari-gaikwad2**

### CODE REVIEW

File: force-app/main/default/classes/CaseTriggerHandler_Test.cls [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+   * @description : this method tests scenario when application is reassigned then the SLA should be reset for the new owner.
+   * @author Ishwari Gaikwad(Thinqloud) | 05-18-2021
+   **/
+  @isTest
+  public static void checkReassignmentForMilestones() {
+    List<Application__c> applicationList = [
+      SELECT Id, Adjudication_Path_picklist__c
+      FROM Application__c
+    ];
+    if (applicationList.size() > 0) {
```
Comment (RemarioCRM) [inline force-app/main/default/classes/CaseTriggerHandler_Test.cls:198]: Please USE Constants in Constants.Cls file in all Tests

**Review by RemarioCRM — CHANGES_REQUESTED** [review]

File: force-app/main/default/classes/CaseTriggerHandler.cls [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+  /**
+   * @description : When Case will be created assign entitlement record to it.
+   * @author Ishwari Gaikwad(Thinqloud) | 05-18-2021
+   **/
+  public static void addEntitlementToCase() {
+    List<Entitlement> entitlementList = [
+      SELECT Id, Name, AccountId
+      FROM Entitlement
+      WHERE Status = 'Active' AND Name = :''
```
Comment (RemarioCRM) [inline force-app/main/default/classes/CaseTriggerHandler.cls:34]: i dont understand this query AND Name = :'', Does the Entitlement not have a name?

**Review by RemarioCRM — CHANGES_REQUESTED** [review]

File: force-app/main/default/classes/CaseTriggerHandler.cls (hunk 2 above)
Comment (ishwari-gaikwad2) [reply force-app/main/default/classes/CaseTriggerHandler.cls:34]: I need to push this file again I have made changes to it. I have added the name in constants and then used that in condition.

File: force-app/main/default/classes/CaseTriggerHandler_Test.cls (hunk 1 above)
Comment (ishwari-gaikwad2) [reply force-app/main/default/classes/CaseTriggerHandler_Test.cls:198]: sure i will use constants in test class as well

**Pull request merged**